rl_loop/selfplay.py

```diff
@@ -51,26 +54,40 @@ def run_cc():
         '--flagfile=rl_loop/distributed_flags'])
 
 
-def run_tpu():
-    mask_flags.checked_run([
+def run_tpu(no_resign=False):
```

Comment:
s/no_resign/calibration/

Reply:
i'd probably like to refactor "No-resign" to "Calibration" in all places instead of being split.